duckduckgo/iOS
# Update iOS privacy defaults

Task/Issue URL: https://app.asana.com/0/72649045549333/1204630981406973/f

**Description**:

- Adds Exclude Local Networks toggle to VPN settings screen
- Updates BSK to include agreed privacy defaults

**Steps to test this PR**:

- Check logs from Console.app while running NetP. Flip the Exclude Local Networks toggle while NetP is/isn't running. The logged excluded routes should contain the corresponding exclusion as per https://app.asana.com/0/0/1205930025614235/f
- Run tests from [DNS Leak Test](https://www.dnsleaktest.com) while NetP is up. The identified server should point to ours. 
- Check IPv6 traffic using https://ipv6-test.com or https://test-ipv6.com. IPv6 should not work while NetP is up.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
